### ISSUE TITLE
scalafix-testkit_2.12.10:0.9.19 not found

### ISSUE THREAD

**Issue by japgolly** [scalacenter/scalafix#1230]
Hello. I'm following the instructions in https://scalacenter.github.io/scalafix/docs/developers/local-rules.html and I'm getting this failure:

```
[warn] 	Note: Unresolved dependencies path:
[error] stack trace is suppressed; run last scalafix-tests / update for the full output
[error] (scalafix-tests / update) sbt.librarymanagement.ResolveException: Error downloading ch.epfl.scala:scalafix-testkit_2.12.10:0.9.19
[error]   Not found
[error]   Not found
[error]   not found: ~/.ivy2/local/ch.epfl.scala/scalafix-testkit_2.12.10/0.9.19/ivys/ivy.xml
[error]   not found: https://repo1.maven.org/maven2/ch/epfl/scala/scalafix-testkit_2.12.10/0.9.19/scalafix-testkit_2.12.10-0.9.19.pom
```

I'm on SBT 1.13.13

**Comment by japgolly** [comment]
Note that SBT 1.3.1 all the way through to 1.3.13 uses Scala 2.12.10.

**Comment by japgolly** [comment]
Btw, I've also got this in my settings:

```scala
ThisBuild / scalafixScalaBinaryVersion := "2.13"
```

**Comment by bjaglin** [comment]
It looks like `scalaVersion` is not explicitly defined in your `scalafix-tests` project. Can it be the same problem as https://github.com/scalacenter/scalafix/issues/1231#issuecomment-679645784 (common settings not applied) ?

In any case, https://scalacenter.github.io/scalafix/docs/developers/local-rules.html should mention that the full Scala version should match the one for which scalafix-testkit is published with so that the https://github.com/scalacenter/scalafix/commit/c0c9e88cbaa432b023ea22025c9f4d9cbb7503fc#diff-d94b0570fdb6cd5564140b6ebf0ca982R188-R192 lookup can work. I'll open a doc PR.

**Comment by japgolly** [comment]
Ahhhhhhhhhhh I thought it wasn't working because the scala version was tied to the SBT plugin. Now that I've fixed my mistake in #1231, re-enabling the tests module works. Thank you for the help @bjaglin!
